File: dataset/output-bk/PERPLEXITY_llama-3-sonar-large-32k-chat/Task10_PERPLEXITY_llama-3-sonar-large-32k-chat.cpp
#include <iostream>

int sumBetween(int a, int b) {
    int min = (a < b) ? a : b;
    int max = (a > b) ? a : b;
    int sum = 0;
    for (int i = min; i <= max; i++) {
        sum += i;
    }
    return sum;
}

int main() {
    std::cout << sumBetween(-1, 2) << std::endl;
    return 0;
}

File: dataset/output-bk/PERPLEXITY_llama-3-sonar-large-32k-chat/Task10_PERPLEXITY_llama-3-sonar-large-32k-chat.java
public class Main {
    public static void main(String[] args) {
        System.out.println(sumBetween(-1, 2));
    }

    public static int sumBetween(int a, int b) {
        int min = Math.min(a, b);
        int max = Math.max(a, b);
        int sum = 0;
        for (int i = min; i <= max; i++) {
            sum += i;
        }
        return sum;
    }
}